src/components/media-player/volume-presets-modal.yml

```diff
@@ -0,0 +1,3 @@
+modalTitle:
+  en: Manage volume presets
+  de: Volumen-Vorwahl bearbeiten
```

Comment:
This one is quite clunky: "Lautstärke-Voreinstellungen bearbeiten"

Reply:
Also in `track-mixer.yml`

Reply:
Indeed, "Lautstärke", not "Volumen"...
Thanks, updated!